python-threatexchange/threatexchange/cli/dataset/simple_serialization.py

```diff
@@ -87,3 +90,61 @@ def load(
                         )
                     )
         return ret
+
+
+class HMASerialization(CliIndicatorSerialization):
+    """
+    A Serialization for HMA Similar to CliIndicatorSerialization but with
+    Indicator ID instead of Descriptor ID
+    """
+
+    def __init__(
+        self,
+        indicator_id: str,
+        indicator_type: str,
+        indicator: str,
+        rollup: SimpleDescriptorRollup,
+    ):
+        self.indicator_id = indicator_id
+        self.indicator_type = indicator_type
+        self.indicator = indicator
+        self.rollup = rollup
+
+    def as_csv_row(self) -> t.Tuple:
+        """indicator details and descriptor rollup without descriptor ID"""
+        return (self.indicator_id, self.indicator) + self.rollup.as_row_without_id()
```

Comment:
nit: to keep it even more consistent could be swap this order of indicator and id? I like the 'signal' itself being the first column personally (not sure if our system actually still depends on it anywhere)